lms/static/sass/_build-lms.scss

```diff
@@ -52,6 +52,7 @@
 @import 'views/decoupled-verification';
 @import 'views/shoppingcart';
 @import 'views/homepage';
+@import 'views/creative-commons';
```

Comment:
I don't see an `lms/static/sass/components` directory. Where should I put the `_creative-commons.scss` file?